docs/model-dev-guide/model-management/checkpoints.rst

```diff
@@ -25,13 +25,13 @@ The Checkpoint Export API is a subset of the features found in the
  Querying Checkpoints
 **********************
 
-The :class:`~determined.experimental.client.Experiment` class is a reference to an experiment. The
-reference contains the :meth:`~determined.experimental.client.Experiment.top_checkpoint` method.
-Without arguments, the method will check the experiment configuration searcher field for the
-``metric`` and ``smaller_is_better`` values. These values are used to sort the experiment's
-checkpoints by validation performance. The searcher settings in the following snippet from an
-experiment configuration file will result in checkpoints being sorted by the loss metric in
-ascending order.
+The :class:`~determined.experimental.client.Experiment` class is a reference to an experiment.
+Within this class, there is a :meth:`~determined.experimental.client.Experiment.list_checkpoints`
+method. When called without arguments, this method looks at the experiment configuration's searcher
+field for the `metric` and `smaller_is_better` values. These values are used to sort the
```

Comment:
this might be an artifact of us being heavily hyperparameter-search oriented... i think making this change is out of scope for this PR since this default comes from the master and i'm not sure how it'd impact other consumers (web) of the API. but yeah i think it's worth asking product/AMLEs.